squirtgun-commands/src/main/java/me/lucyy/squirtgun/command/node/subcommand/SubcommandNode.java

```diff
@@ -35,15 +36,14 @@
 import org.jetbrains.annotations.Nullable;
 
 import java.util.*;
```

Comment:
Absolutely no wildcard imports.
https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports
https://stackoverflow.com/q/19118563/8083725
https://stackoverflow.com/q/147454/8083725
https://softwareengineering.stackexchange.com/q/375901

It isn't hard nowadays to just let the editors and tools to generate the individual imports for us.

Reply:
oops, fixed in 38c08eb